workflow/rocoto/sfs_tasks.py

```diff
@@ -0,0 +1,632 @@
+from applications.applications import AppConfig
+from rocoto.tasks import Tasks
+import rocoto.rocoto as rocoto
+
+
+class SFSTasks(Tasks):
+
+    def __init__(self, app_config: AppConfig, run: str) -> None:
+        super().__init__(app_config, run)
+
+    def stage_ic(self):
+
+        resources = self.get_resource('stage_ic')
+        task_name = f'{self.run}_stage_ic'
+        task_dict = {'task_name': task_name,
+                     'resources': resources,
+                     'envars': self.envars,
+                     'cycledef': f'{self.run}',
+                     'command': f'{self.HOMEgfs}/jobs/rocoto/stage_ic.sh',
+                     'job_name': f'{self.pslot}_{task_name}_@H',
+                     'log': f'{self.rotdir}/logs/@Y@m@d@H/{task_name}.log',
+                     'maxtries': '&MAXTRIES;'
+                     }
+        task = rocoto.create_task(task_dict)
+
+        return task
+
+    def waveinit(self):
+
+        resources = self.get_resource('waveinit')
+        task_name = f'{self.run}_wave_init'
```

Comment:
```suggestion
        task_name = f'{self.run}_waveinit'
```

Reply:
`sfs_tasks.py` is literally a direct copy of `gefs_tasks.py` with the class name changed.

Reply:
Works for me.  Thanks.